app/models/user.rb

```diff
@@ -35,6 +35,7 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :trackable, :validatable
   include DeviseTokenAuth::Concerns::User
+  include Flipper::Identifier
```

Comment:
What is this for? Just asking.

Reply:
I think it's to add the `flipper_id` method that identifies the "actor".

Should we include it in `ApplicationRecord` so that all of our records have the method defined? For example an app could have a model `Product` and we might only want to enable certain feature for some products.. just thinking out loud, would like to hear others' thoughts

Reply:
I think that could be an option, the implementation shouldn't impact or be tightly coupled with the business logic

Reply:
I'm not sure about including this in `ApplicationRecord`. Feature flags are typically required for only a limited number of models, like User, Organization, Product.

Reply:
Can you check if [this](https://github.com/jnunemaker/flipper/blob/a82aa8d93c43725a98219ad3b61160ebf65e9cc4/lib/flipper/railtie.rb#L32-L36) is true? Seems like Flipper already configures `ActiveRecord::Base` to include `Flipper::Identifier`

Reply:
@santib yes, nice catch, it looks like their getting started guide is out of date